js/settings.js

```diff
@@ -299,4 +299,23 @@ settings.define_upgrade_callback(32, function(s) {
   // fix typo in query definition
   s.saves["MapCSS styling"].overpass = s.saves["MapCSS styling"].overpass.replace("<;",">;");
   s.save();
-});
\ No newline at end of file
+});
+
+settings.define_upgrade_callback(33, function(s) {
+  s.saves["Attic date query"] = {
+    type: "example",
+    overpass: [
+      '/* This query loads all objects as of 2016-01-01 */',
+      '[date:"2016-01-01T00:00:00Z"]',
```

Comment:
fixed in 1b2033bf9dac04b1ef8dea74302a67252fd5104e and published on overpass-turbo.eu